### PR TITLE
MN-455: implement token auth for observer

### PR DESCRIPTION
**Pull request by MikhailPopov**
**- What I did**
- I have expanded the configuration of the replicator (some parameters like Auth.RefreshOffset and InsecureTLS are useful for testing).
- I have implemented a PerRPCCredential interface, in which we check a token and receive a new one if needed.
- I have removed two old not used files.

### CODE REVIEW

File: connectivity/connectivity.go [hunk 1]
```diff
@@ -32,8 +37,26 @@ func Make(cfg *configuration.Observer, obs *observability.Observability) *Connec
 			)
 			log.Infof("trying connect to %s...", cfg.Replicator.Addr)
 
-			// We omit error here because connect happens in background.
-			conn, err := grpc.Dial(cfg.Replicator.Addr, limits, grpc.WithInsecure())
+			cp, err := x509.SystemCertPool()
+			if err != nil {
+				log.Fatal(errors.Wrapf(err, "failed get x509 SystemCertPool"))
+			}
+			httpClient := &http.Client{Transport: &http.Transport{
```
Comment (uhzxl) [inline connectivity/connectivity.go:44]: Maybe we should set up `http.Client` and `http.Transport` timeouts to prevent idle connections?

Comment (MikhailPopov) [reply connectivity/connectivity.go:44]: Good idea. What do you think, should we pass these parameters through a configuration?

Comment (uhzxl) [reply connectivity/connectivity.go:44]: Yeap, I think so.

Comment (MikhailPopov) [reply connectivity/connectivity.go]: I've added a common timeout for an HTTP client.

**Pull request merged**